app/models/conference.rb

```diff
@@ -16,4 +16,10 @@ class Conference < ApplicationRecord
   def self.ransackable_attributes(_auth_object = nil)
     %w[name]
   end
+
+  def session_dates
```

Comment:
I suggest removing the cache. The query seems to be very fast


<img width="1436" alt="image" src="https://github.com/user-attachments/assets/668c5f55-816b-4f64-a62e-06523174609c">